molecule/command/base.py

```diff
@@ -122,6 +123,31 @@ def execute_cmdline_scenarios(scenario_name, args, command_args, ansible_args=()
                 raise
 
 
+def _output_for_ci(func):
+    is_travis = os.getenv("TRAVIS") and os.getenv("CI")
+    if not is_travis:
+        return func
+
+    @functools.wraps(func)
+    def travis_wrapper(config, subcommand):
+        print(
+            (
+                "travis_fold:start:{0}.{1}"
+                "\033[33;1mMolecule Scenario: '{0}' Action: '{1}'\033[0m"
+            ).format(
+                config.scenario.name,
+                subcommand,
+            )
```

Comment:
```suggestion
            (
                "travis_fold:start:{0}.{1}"
                "{2}Molecule Scenario: '{0}' Action: '{1}'{3}"
            ).format(
                config.scenario.name,
                subcommand,
                colorama.Style.Bold + colorama.Fore.CYAN,
                colorama.Style.RESET_ALL,
            )
```

We might also be able to use `colorama` like in logger.py (would need to test to make sure the escape sequence works in this context (colorama uses `\x1b` fwict but the example I found uses `\033`).